src/views/primary-auth/PrimaryAuthForm.js

```diff
@@ -167,8 +173,9 @@ define([
       } else if (!this.settings.get('features.passwordlessAuth')) {
         this.getInputs().toArray()[1].focus();
       }
+      if (this.settings.get('features.trackTypingPattern')) {
+        TypingUtil.track('okta-signin-username');
```

Comment:
just checking: if that selector id does not exist in the dom, it returns null right? so all works fine in that case.

Reply:
If Id is not valid it tracks only clicks and special character press. Doest really stop authn though.

https://github.com/TypingDNA/TypingDnaRecorder-JavaScript#typingdnaaddtargetelement_id

Reply:
ok... we can keep an eye on this. Otherwise we can move this logic to line 172, inside the `if` block for when username is empty and the user needs to type it.